### PR TITLE
added routes

### PR DESCRIPTION
**Pull request by seanwayland**
I added a simple homepage and an album page with a react router

### CODE REVIEW

File: .idea/frontend.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ctsexton) [inline .idea/frontend.iml:1]: Please add `.idea` to the gitignore so we don't have these files (intellijel config files?) accidentally added to commits.

File: src/App.js [hunk 2]
```diff
@@ -1,92 +1,33 @@
...
+                            <li><Link to={'/Album'} className="nav-link">Album</Link></li>
+
+
+                        </ul>
+                    </nav>
+                    <hr />
+                    <Switch>
+                        <Route exact path='/' component={Home} />
+
+                        <Route path='/Album' component={Album} />
```
Comment (ctsexton) [inline src/App.js:25]: Nice

File: src/Home.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class Home extends Component {
+    render() {
+        return (
+
+            <div>
+
+                <img src="https://waylostreams.live/waylostreams.jpg"></img>
```
Comment (ctsexton) [inline src/Home.js:9]: This really is Waylostreams now!

File: .idea/frontend.iml (hunk 1 above)
Comment (seanwayland) [reply .idea/frontend.iml]: Ok I think I just did that

**Pull request merged**